packaging/distros/ubuntu/Dockerfile

```diff
@@ -39,7 +42,8 @@ RUN apt-get update && \
     cmake make bash wget unzip nano vim valgrind dh-make flex bison \
     libpq-dev postgresql-server-dev-all \
     libsasl2-2 libsasl2-dev openssl libssl-dev libssl1.1 \
-    software-properties-common libyaml-dev apt-transport-https pkg-config zlib1g-dev && \
+    software-properties-common libyaml-dev apt-transport-https pkg-config zlib1g-dev \
+    llvm-10 libclang-common-10-dev libedit-dev && \
```

Comment:
We'll plan to move LLVM 13 or later on the PR https://github.com/fluent/fluent-bit/pull/7587.
If you could, could you investigate and reflect to use LLVM 13 or later, please?
For Ubuntu, we can introduce LLVM 13 or later with the aptline:
https://apt.llvm.org/

Reply:
Dropping flb-wamrc compilation for Ubuntu.16.04 would be reasonable, I guess.